tasks/component-usage-graph.js

```diff
@@ -111,39 +115,42 @@ export function getDependantsLsStyle (componentPath, graph, maxLevel = Infinity)
 
 /**
  *
- * @param inputs
- * @param graph
- * @param mode
- * @param maxLevel
+ * @param {string[]} componentsPath
+ * @param {DepGraph} graph
+ * @param {'dependencies'|'dependants'} mode
+ * @param {number} [maxLevel=Infinity]
  * @returns {string|null}
+ *
+ * This function allows you to get either the dependencies or the dependants of one or more components given its path list.
+ * It will work whether you provide it with an array of one component path or a full list.
```

Comment:
fix: In JSDoc, the description of the function is always above @param and @return tags.